packages/gestalt/src/Popover/Controller.js

```diff
@@ -0,0 +1,93 @@
+// @flow strict
```

Comment:
Copied legacy Controller and converted to function component removing unecessary parts.